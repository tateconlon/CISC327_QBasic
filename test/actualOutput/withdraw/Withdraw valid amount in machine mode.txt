login as machine successful
Account 1234567 does not exist
transaction code 500 invalid
Account 1234567 does not exist
transaction code 100 invalid
Transaction summary written and system logged out successfully