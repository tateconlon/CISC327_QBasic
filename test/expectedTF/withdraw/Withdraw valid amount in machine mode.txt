DEP 1234567 500 *** ***
WDR 1234567 100 *** ***
EOS