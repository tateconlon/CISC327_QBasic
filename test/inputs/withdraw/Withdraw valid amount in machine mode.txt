login
machine
deposit
1234567
500
withdraw
1234567
100
logout