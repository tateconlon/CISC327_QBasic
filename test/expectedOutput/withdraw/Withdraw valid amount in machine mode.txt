login as machine successful
deposit 1234567 500 successful
withdraw 1234567 100 successful
Transaction summary written and system logged out successfully